bindings/wasm/src/account/types/signature.rs

```diff
@@ -9,8 +9,10 @@ use wasm_bindgen::prelude::*;
 
 use crate::error::wasm_error;
 use crate::error::Result;
+use crate::error::WasmResult;
 
 #[wasm_bindgen(js_name = Signature, inspectable)]
+#[derive(Deserialize)]
 pub struct WasmSignature(pub(crate) Signature);
```

Comment:
Can we investigate whether we actually need a custom `Account` signature? From what I see we don't use the returned public key, only the signature data vector.